### PR TITLE
feat(cli): add welcome message

### PR DESCRIPTION
**Pull request by c3ppc3pp**

### CODE REVIEW

File: golang/pkg/cli/config_file.go [hunk 1]
```diff
@@ -307,6 +308,13 @@ func DisabledServiceSettings(settings *Settings) *Settings {
 func PrintInfo(settings *Settings) {
 	log.Warn().Msg("🦩🦩🦩 Use the CLI tool only for NON-PRODUCTION purpose. 🦩🦩🦩")
 
+	if !settings.SettingsFile.DisableWelcomeMessage {
+		log.Info().Msgf("The config file is located at %s, where you can turn this message off.", settings.SettingsFilePath)
+		log.Info().Msgf("If you have any questions head to our Discord - discord.gg/pZWbd4fxga ! We're happy to help!")
+		log.Info().Msgf("You can learn more about the project at docs.dyrector.io, if you found this project useful please " +
+			"give us a star - github.com/dyrector-io/dyrectorio")
```
Comment (nandor-magyar) [inline golang/pkg/cli/config_file.go:315]: `https://`  before link makes it clickable I think.

File: golang/pkg/cli/config_file.go [hunk 2]
```diff
@@ -307,6 +308,13 @@ func DisabledServiceSettings(settings *Settings) *Settings {
 func PrintInfo(settings *Settings) {
 	log.Warn().Msg("🦩🦩🦩 Use the CLI tool only for NON-PRODUCTION purpose. 🦩🦩🦩")
 
+	if !settings.SettingsFile.DisableWelcomeMessage {
```
Comment (nandor-magyar) [inline golang/pkg/cli/config_file.go:311]: Can we just make this happen once?

File: golang/pkg/cli/config_file.go (hunk 1 above)
Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: Added.

File: golang/pkg/cli/config_file.go (hunk 2 above)
Comment (nandor-magyar) [reply golang/pkg/cli/config_file.go:312]: Discussed in person. Output: too much chattiness is undesired by us and possibly by others as well.

Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: Fixed.

File: golang/pkg/cli/config_file.go [hunk 3]
```diff
@@ -324,47 +327,62 @@ func PrintInfo(settings *Settings) {
...
-				err = os.Mkdir(userConfDir, DirPerms)
-				if err != nil {
-					log.Fatal().Err(err).Stack().Send()
-				}
-			} else if err != nil {
+	userConfDir, err := os.UserConfigDir()
+	if err != nil {
+		log.Fatal().Err(err).Stack().Send()
+	}
+	settingspath := SettingsPath()
```
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: image

File: golang/pkg/cli/config_file.go [hunk 4]
```diff
@@ -324,47 +327,62 @@ func PrintInfo(settings *Settings) {
...
+
+	// If settingspath is default, we create the directory for it
+	if settings.SettingsFilePath == settingspath {
```
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: fixed

File: golang/pkg/cli/config_file.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: fixed

File: golang/pkg/cli/config_file.go [hunk 5]
```diff
@@ -324,47 +327,62 @@ func PrintInfo(settings *Settings) {
 	log.Info().Msg("Happy deploying! 🎬")
 }
 
+func SettingsPath() string {
+	userConfDir, err := os.UserConfigDir()
+	if err != nil {
+		log.Fatal().Err(err).Stack().Send()
+	}
+
+	return fmt.Sprintf("%s/%s/%s", userConfDir, SettingsFileDir, SettingsFileName)
```
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply golang/pkg/cli/config_file.go]: fixed

**Review by nandor-magyar — APPROVED** [review]
LGTM! :+1:

**Pull request merged**